docs/HowTo/clusterconnectivity/aiven-kafka-cluster-sasl-ssl-protocol.md

```diff
@@ -20,16 +20,16 @@ Apache Kafka® cluster with Klaw using the SASL_SSL authentication protocol.
 ## Configure and connect using SASL protocol
 
 Follow the steps below to configure and connect to an Aiven for Apache
-Kafka® and Klaw using SSL protocol:
+Apache Kafka® and Klaw using SSL protocol:
 
 1. In the Klaw web interface, navigate to **Environments**, and click
    **Clusters**.
 2. On the **Clusters** page, click **Add Cluster**.
-3. On the **Add Kafka cluster** page, enter the following details:
-   - **Cluster Type:** Select **Kafka** from the drop-down list
+3. On the **Add Apache Kafka cluster** page, enter the following details:
+   - **Cluster Type:** Select **Apache Kafka** from the drop-down list
```

Comment:
Same as above, the mention of kafka in the bold part is a UI description. To make vale handle that correctly:

```suggestion

   <!-- vale off -->
   - **Cluster Type:** Select **Kafka** from the drop-down list
   <!-- vale on -->
```